kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiReservedKeys.scala

```diff
@@ -19,6 +19,7 @@ package org.apache.kyuubi.config
 
 object KyuubiReservedKeys {
   final val KYUUBI_SESSION_USER_KEY = "kyuubi.session.user"
+  final val KYUUBI_SESSION_IP_KEY = "kyuubi.session.ip"
```

Comment:
we have `kyuubi.client.ip` already, how about using this?

maybe  kyuubi.client.ipAddress is better

Reply:
thanks for the information